### ISSUE TITLE
vagrant up --provision: ssh connection fails

### ISSUE THREAD

**Issue by capsular** [geerlingguy/drupal-vm#1775]
### Issue Type
Support Request

### Your Environment
```
Vagrant 2.1.1
VirtualBox 5.2.12r122591
ansible 2.2.0.0
  config file = 
  configured module search path = Default w/o overrides
```

### Your OS
 - **macOS** (10.13.4)

### Full console output

```
vagrant up --provision
...
...
TASK [setup] *******************************************************************
fatal: [drupalvm]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 127.0.0.1 port 2222: Operation timed out\r\n", "unreachable": true}
```

### Summary
I tried all solutions I found in related issues without success (vagrant box remove and than re-add, deleting curl). Also upgrading vitualbox, and Ansible didn't help. As I don't think I can solve the issue I wonder if there's a way to get to the database files of my domains.

**Comment by capsular** [comment]
Sorry, hit enter by accident and published too early.. I tried to log in to the geerlingguy/ubuntu box and the drupalvm.tes box through VirtualBox GUI but can't find any database files there..

**Comment by geerlingguy** [comment]
If that first `setup` task fails, that means none of the Drupal VM provisioning happened at all, which is why there's nothing (databases, Apache, etc.) inside the VM.

If Vagrant can't connect Ansible into the VM, then something with the connection seems to be breaking. Do you have any customizations in a `config.yml`? Are you using a VPN on your computer, or do you have any kind of virus scanning or network filtering software?

Can you also (on your Mac) run the command `cat .vagrant/provisioners/ansible/inventory`, or run `vagrant ssh-config` to see the SSH connection details?

For some reason Ansible is not able to connect to your VM.

**Comment by capsular** [comment]
Thanks so much for your reply! 

vagrant ssh-config
Host drupalvm
  HostName 127.0.0.1
  User vagrant
  Port 2222
  UserKnownHostsFile /dev/null
  StrictHostKeyChecking no
  PasswordAuthentication no
  IdentityFile /Users/andremu/.vagrant.d/insecure_private_key
  IdentitiesOnly yes
  LogLevel FATAL
  ForwardAgent yes

before the problems occurred I ran the root password reset script for one of the websites in my local drupalvm folder (drupal 7, ./scripts/password-hash.sh) but I guess that has nothing to do with this..

I ran vagrant up --provision again and got

==> drupalvm: Waiting for machine to boot. This may take a few minutes...
    drupalvm: SSH address: 127.0.0.1:2222
    drupalvm: SSH username: vagrant
    drupalvm: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.
...
no more info on why connection failed..
couldn't it be that the .vmdk is corrupt?
